src/AElf.Kernel.Types/Block/Block.cs

```diff
@@ -25,6 +26,8 @@ public Block(Hash preBlockHash)
             Body = new BlockBody();
         }
 
+        public IEnumerable<Hash> TransactionList => Body.Transactions;
```

Comment:
rename to TransactionHashList

Reply:
for the TODO, opened a tracking issue : #1692